package.json

```diff
@@ -17,8 +17,9 @@
     "lint:clang": "./scripts/run-clang-format.sh",
     "lint:swift": "./scripts/run-swiftlint.sh",
     "test": "jest",
+    "android:test": "npx lerna run android:test",
     "ios:install": "npx lerna run ios:install",
-    "ios:test": "npx lerna run ios:test"
+    "ios:test": "npx lerna run ios:test --concurrency=1"
```

Comment:
What does this concurrency ? Prevent run more than 1 test at the same time ?

Reply:
it prevents more than one package running `ios:test` at the same time, I was finding when core + this one ran their ios tests together that there were failures I guess due to the simulator on the github runner trying to do two things at once and/or too many resources being used

Reply:
I see, I think this does the same that maxInstances: 1 in capabilities inside the wdio config. 
```

{
      platformName: "iOS",
      "appium:automationName": "XCUITest",
      "appium:deviceName": "iPhone 16 Pro",
      "appium:bundleId": "io.embrace.basictestapp",
      "appium:noReset": true,
       maxInstances: 1,
     }
```

Reply:
ya similar idea, this is running the swift unit tests though not the integration tests through web driver those are still not part of the GHA workflows

Reply:
oh I see